### PR TITLE
Create intital Login Page (no functionality)

### PR DESCRIPTION
**Pull request by MerwinAamir1**
Screenshot 2024-09-22 at 9 21 04 PM

Created initial design of LoginPage. Keep in mind we should use firebase for login authentication but this is just the design concept. Once we are onboarded to firebase we can continue with this.

### CODE REVIEW

**Comment by MerwinAamir1** [comment]
One thing to note: I created the src and screens directory. Lets keep everything by maintaining this structure. If we want to make a reuseable component we should make a Components folder and put it there.

**Comment by MerwinAamir1** [comment]
Another thing Is we most likely want to use a css framework for future css. Here I used a plain stylesheet. Once that is determined I can migrate to that.

**Comment by maryjabro** [comment]
I like it! We should have some VitalTrack Logo on the sign in page. This is a good base

File: VitalTrackApp/src/screens/Login.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+export default function LoginScreen() {
+  return (
+    <View style={styles.container}>
+      <Text style={styles.title}>Welcome Back</Text>
```
Comment (maryjabro) [inline VitalTrackApp/src/screens/Login.js:11]: Preferably having  "VitalTrack" text above welcome

**Review by maryjabro — APPROVED** [review]

File: VitalTrackApp/src/screens/Login.js (hunk 1 above)
Comment (MerwinAamir1) [reply VitalTrackApp/src/screens/Login.js:11]: Sure I will address this

**Pull request merged**